internal/controllers/reports.go

```diff
@@ -52,6 +81,19 @@ func (r *ReportsServer) GetMemberCounts(w http.ResponseWriter, req *http.Request
 	ok(w, charts)
 }
 
+func (r *ReportsServer) GetMemberChurn(w http.ResponseWriter, req *http.Request) {
+	churn, err := r.store.GetMemberChurn()
+	if err != nil {
+		log.Errorf("error getting member churn: %s", err)
+		http.Error(w, "error getting member churn", http.StatusInternalServerError)
+		return
+	}
```

Comment:
line 85 - 90 should probably be inside a service layer. Might be worth pulling in `main` to get the existing service for member and used that to create the report service